### DISCUSSION TITLE
how can we get geology data

### DISCUSSION THREAD

**Discussion by youzhw-HUST** [mhm-ufz/mHM#79] [Q&A]

**Comment by pallavhydro** [comment]
Hi @youzhw-HUST 


One option for geological data can be the GLiM. 
Here is the link to the GLiM dataset at the PANGEA database:
https://doi.pangaea.de/10.1594/PANGAEA.788537


Hope this helps
Pallav

**Reply by youzhw-HUST** [reply]
Much thanks for your sharing! I'll have some try on this.

**Reply by pallavhydro** [reply]
Hi @youzhw-HUST 

If the discussion here solved your issue, I'd appreciate you mark the response that solved the issue.

Thanks in advance.
Pallav
